addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -368,39 +368,39 @@ def test_mailing_unique_name(self):
         If the name is missing, it's generated from the subject. Then we should ensure
         that this generated name is unique.
         """
-        mailing_0 = self.env['mailing.mailing'].create({'subject': 'First subject'})
+        mailing_0 = self.env['mailing.mailing'].create({'subject': 'First subject0'})
 
         mailing_1, mailing_2, mailing_3, mailing_4, mailing_5, mailing_6 = self.env['mailing.mailing'].create([{
-            'subject': 'First subject',
+            'subject': 'First subject1',
         }, {
-            'subject': 'First subject',
+            'subject': 'First subject2',
         }, {
             'subject': 'First subject',
             'source_id': self.env['utm.source'].create({'name': 'Custom Source'}).id,
         }, {
             'subject': 'First subject',
-            'name': 'Mailing',
+            'name': 'Mailing1',
         }, {
             'subject': 'Second subject',
-            'name': 'Mailing',
+            'name': 'Mailing2',
```

Comment:
Overall I don't think it makes sense to change this test. Either we do want the mailings to have different names for some reason (I don't think so, after this change).

If the intent of the change is "you should ensure the name is unique yourself", then we should remove duplicates of the same day from the test, as it's now an invalid input.